tests/test_geoprocessing.py

```diff
@@ -5682,6 +5682,31 @@ def test_get_raster_info_overviews(self):
         self.assertEqual(
             raster_info['overviews'], [(500, 1000), (250, 500)])
 
+    def test_get_raster_info_vsicurl(self):
+        """PGP: raster info with vsicurl."""
+        # Access test data hosted on github
+        path = '/vsicurl/https://raw.githubusercontent.com/emlys/pygeoprocessing/feature/441/tests/test_data/raster.tif'
+        raster_info = pygeoprocessing.get_raster_info(path)
+        self.assertEqual(raster_info['raster_size'], (9, 9))
+
+    def test_get_vector_info_vsicurl(self):
+        """PGP: vector info with vsicurl."""
+        # Access test data hosted on github
+        path = '/vsicurl/https://raw.githubusercontent.com/emlys/pygeoprocessing/feature/441/tests/test_data/vector.gpkg'
+        vector_info = pygeoprocessing.get_vector_info(path)
+        self.assertEqual(vector_info['bounding_box'], [-15, 14.8, -14.8, 15])
+
+    def test_raster_map_vsicurl(self):
+        """PGP: raster_map with vsicurl."""
+        # Access test data hosted on github
+        input_path = '/vsicurl/https://raw.githubusercontent.com/emlys/pygeoprocessing/feature/441/tests/test_data/raster.tif'
+        target_path = os.path.join(self.workspace_dir, 'out.tif')
+        pygeoprocessing.raster_map(
+            lambda block: block * 2, [input_path], target_path)
+        numpy.testing.assert_array_equal(
+            pygeoprocessing.raster_to_numpy_array(target_path),
+            numpy.ones((9, 9), dtype=numpy.float32))

```

Comment:
This got me thinking that it'd be nice to document what `test_data/raster.tif` is and it's properties. Maybe a more descriptive name for that test file could help at the very least. But if I wanted to use it in another test function I wouldn't know what to expect for values.